packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -1400,12 +1400,15 @@ export class DataViewExpander {
         const matchedAttributeDTOs = await this.consumption.attributes.executeIdentityAttributeQuery({
             query
         });
-        const matchedAttributeDVOs = await this.expandLocalAttributeDTOs(matchedAttributeDTOs.value);
+        if (matchedAttributeDTOs.isError) throw matchedAttributeDTOs.error;
+
+        const matchedAttributeDTOsSortedByDefaultFirst = matchedAttributeDTOs.value.sort((attr1, attr2) => Number(!!attr2.isDefault) - Number(!!attr1.isDefault));
```

Comment:
I don't know where this started, but please don't use `Number()` or `String()` constructors.

I'd write this as 
```suggestion
        const matchedAttributeDTOsSortedByDefaultFirst = matchedAttributeDTOs.value.sort((attribute1, attribute2) =>
            attribute1.isDefault === attribute2.isDefault ? 0 : attribute1.isDefault ? 1 : -1
        );
```

also please see attr1 -> attribute1, I don't see a reason for abbreviations here.

Reply:
urgh, I fd up :D 

`attribute1.isDefault ? 1 : -1` should be `attribute1.isDefault ? -1 : 1` or it gets sorted the wrong way.

Reply:
you can try it out with this js / ts script

```ts
const value = [
    {
        isDefault: false
    },
    {
        isDefault: true
    },
    {
        isDefault: false
    },
    {
        isDefault: undefined
    }
];

const matchedAttributeDTOsSortedByDefaultFirst = value.sort((attribute1, attribute2) => (attribute1.isDefault === attribute2.isDefault ? 0 : attribute1.isDefault ? -1 : 1));

console.log(matchedAttributeDTOsSortedByDefaultFirst);
```